src/content.ts

```diff
@@ -74,7 +78,10 @@ export const reply = async (
   const currentFromId = config.requireGetCurrentFromId(opts);
 
   const contentHash = keccak256(content);
-  const uri = await store.put(contentHash, content, opts);
+  const store = requireGetStore(opts);
+  const uri = await store.putStream(contentHash, async (stream: WriteStream) => {
```

Comment:
randomFilename
uuid?
`uuid-timestamp-contentHash`

Reply:
Let's discuss, but not change in this PR.